src/functions-templates/javascript/scheduled-function/{{name}}.js

```diff
@@ -0,0 +1,12 @@
+const { schedule } = require('@netlify/functions')
+
+const formatAsDateTime = (date) => `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`
+
+module.exports.handler = schedule('* * * * *', async () => {
+  const currentExecutionTime = formatAsDateTime(new Date())
+  console.log(`Function executed at ${currentExecutionTime}.`)
```

Comment:
[dust] Looks good! I would use `Function executed at ${new Date()}`, without the conversion. I'd expect the users to delete the code pretty much immediately and adjust to the functions _they_ want to write, so it would be great for them to have to delete less code

Reply:
Agreed!

Reply:
This makes sense. I've applied the changes to  theJS & TS templates.